server/constants/messages.go

```diff
@@ -2,30 +2,45 @@ package constants
 
 const (
 	// TODO: all these messages are to be verified from Mike at the end
-	GenericErrorMessage             = "Something went wrong, please try again later"
+	// Generic
+	GenericErrorMessage  = "Something went wrong, please try again later"
+	ConnectAccount       = "[Click here to link your Azure DevOps account](%s%s)"
+	ConnectAccountFirst  = "You do not have any Azure Devops account connected. Kindly link the account first"
+	UserConnected        = "Your Azure Devops account is succesfully connected!"
+	UserAlreadyConnected = "Your Azure Devops account is already connected"
+	UserDisconnected     = "Your Azure Devops account is now disconnected"
```

Comment:
```suggestion
	UserAlreadyConnected = "Your Azure DevOps account is already connected."
	UserDisconnected     = "Your Azure DevOps account is now disconnected."
```
You should change all references of `Devops` to `DevOps`.